web/src/containers/Trade/index.js

```diff
@@ -537,6 +537,7 @@ class Trade extends PureComponent {
 							pairData={pairData}
 							pair={pair}
 							tool={key}
+							ticker={this.props.ticker}
```

Comment:
We can drop this after lifting the logic up to the Trade index. See the comment on the trade block for more details.